src/app/root-store/server-data/vm-snapshots/vm-snapshots.state.ts

```diff
@@ -1,12 +1,19 @@
 import { createEntityAdapter, EntityAdapter, EntityState } from '@ngrx/entity';
 import { VmSnapshot } from '../../../shared/models/vm-snapshot.model';
+import moment = require('moment');
```

Comment:
`require` is a more nodejs way to import dependency. I prefer to use `import * as moment from 'moment'`

Reply:
You increasing coupling with `moment` by injecting it directly into this class. For example, we prefer to encapsulate the logic related to dates inside of some kind of `DatesService` and use it inside of our code. It would allow us to change `moment` to any other library which works with dates without any additional changes in other parts.